CutVariations/runCutVar.sh

```diff
@@ -0,0 +1,79 @@
+# \author Alice Caluisi <alice.caluisi@cern.ch>
+# \since December 2023
+
+#
+# This task performs cut variation studies on trees analysis results using the configuration produced by configCutVar.sh
+# The produced results are called AnalysisResults_cutName.root and are stored in Results/LHC_Test/CutVariations folder 
+#
+# To run the task, type: $> bash runCutVar.sh
+
+#!/bin/bash
+
+reset
+CfgDir=$(dirname "$0")
+Cfg="-b"
+
+#Define an array of all the cuts
+cuts=(
+    "maxChi2PerClusterITS=30"
+    "maxChi2PerClusterITS=40"
+    "maxChi2PerClusterTPC=6"
+    "maxChi2PerClusterTPC=4"
+    "globalTrack=true"
+    "minPt=0.2"
+)
+
+#Call the configCutVar.sh script to generate the configuration file
+bash configCutVar.sh "${cuts[@]}"
+
+#Create subdirectory for cut variation studies in Results/LHC_Test folder
+Results="../Results/LHC_Test/"
+mkdir -p "${Results}/CutVariations/" 
+
+#Generate the list for the config file
+echo "${Results}AnalysisResults.root" > list.txt 
+
+Cfg="--configuration json://Configurations/cutvar_config.json -b" #This is the configuration produced by configCutVar.sh
+
+function runSpec {
+    local cutSpec=$1
+
+    #If globalTrack=true, set customTrack=false
+    if [[ "$cutSpec" == "globalTracktrue" ]]; then
+        sed -i 's/"customTrack": "true"/"customTrack": "false"/' Configurations/cutvar_config.json
+    fi
+
+    #If the cutSpec is a float, remove '.' in the file name
+    if [[ "$cutSpec" == "minPt0.2" ]]; then
```

Comment:
Maybe you can search and replace only the `.`  to generalize it for let's say three different minimum pt cases.
Meaning that you have one if statement like this that would apply to three minimum pT values in your cuts.